src/features/user/UserProfile/components/AddTargetModal.tsx

```diff
@@ -67,9 +69,14 @@ export default function AddTargetModal({
               defaultValue={userprofile.score.target ? userprofile.score.target : null}
               onChange={(e) => setTarget(e.target.value)}
               variant="outlined"
+              inputRef={register({
+                min: 0
```

Comment:
The backend declines zero as a possible target - so either the backend or the client should be changed:

<img width="1259" alt="Bildschirmfoto 2020-11-26 um 13 36 48" src="https://user-images.githubusercontent.com/1532418/100351890-c0661600-301e-11eb-97c4-597a86fe4f9b.png">

Reply:
Another issue: When I enter -100 as a value, I get this view with the string "invalid" after I clicked on the save button before the dialog closed automatically after the API call failed and my target does not get changed: 
<img width="305" alt="Bildschirmfoto 2020-11-26 um 13 35 06" src="https://user-images.githubusercontent.com/1532418/100352434-8e08e880-301f-11eb-9d48-2d9bc368957f.png">

Reply:
I would suggest to show the user an error message if the API call fails and the target can not be changed.